script.js

```diff
@@ -1,4 +1,171 @@
+/*
+How many hours did you spend on this assignment?:
+4 hours +
+
+What part of the assignment did you spend the most time on?:
+Making sure each function worked well
+
+How comfortable did you feel with this assignment? (1-5):
+4
+
+Is there anything in this code that you feel pleased about?:
+Breaking up the game into functions then reading the winner or loswer based on function outputs of each player
+
+What's one aspect of your code you would like specific, elaborate feedback on?:
+Is it ok? Not really sure what is considered good code, or if anything that could be improved on.
+
+*/
+var gameState = "pageLoad";
+var player1DiceRoll = [];
+var player2DiceRoll = [];
+var player1TotalScore = 0;
+var player2TotalScore = 0;
+var player1TotalWins = 0;
+var player2TotalWins = 0;
+var player1Name = "Player 1";
+var player2Name = "Player 2";
+var myOutputValue = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // Naming players
+  if (gameState == "pageLoad") {
+    gameState = "firstClick";
```

Comment:
Is this `if` statement necessary? Can you just start with `gameState = firstClick`?